### ISSUE TITLE
object detection quantized models

### ISSUE THREAD

**Issue by aminsabetsarvestani** [pytorch/pytorch#31316]
Any sample code of object detection quantization?
 I got the following errors when I try to quantized the faster-rcnn. 

Could not run 'aten::empty_strided' with arguments from the 'QuantizedCPUTensorId' backend. 'aten::empty_strided' is only available for these backends: [CPUTensorId, VariableTensorId]

Could not run 'aten::div.Tensor' with arguments from the 'QuantizedCPUTensorId' backend. 'aten::div.Tensor' is only available for these backends: [CPUTensorId, SparseCPUTensorId, VariableTensorId].

**Comment by raghuramank100** [comment]
We do not have object detection tutorials yet, however you can quantize the feature extraction trunk using the currently available tools.  We will add support for more operators and have an example tutorial for object detection in the next few months.

**Comment by aminsabetsarvestani** [comment]
@raghuramank100,  Thanks for your reply and I hope you provide the object detection tutorial soon.  For the quantization, Pytorch uses QuantizedCPUTensorId backend, so I can't jointly process the quantize part of the object detection model (the feature extraction trunk) with the other parts of the network which use different backends.  I can only quantize and investigate the feature extraction trunk of the object detection models, separately. Is it possible to use different backends for different parts of the network?

**Comment by gottbrath** [comment]
Is the model you are trying to quantize: https://github.com/pytorch/vision/blob/master/torchvision/models/detection/faster_rcnn.py

**Comment by aminsabetsarvestani** [comment]
> Is the model you are trying to quantize: https://github.com/pytorch/vision/blob/master/torchvision/models/detection/faster_rcnn.py

Yes, This is the model. I got the mentioned errors in the  RegionProposalNetwork part of the network when calling the box_coder.decode(pred_bbox_deltas.detach(), anchors) function.

**Comment by raghuramank100** [comment]
@aminsabetsarvestani : You should be able to quantize only the feature extraction trunk and keep the rest of the model in float.

**Comment by raviv** [comment]
I'm trying to quantize `torchvision.models.detection.fasterrcnn_resnet50_fpn` to increase inference speed on GPU.
Could you help with a code sample for doing that?
Thanks.

**Comment by vkuzo** [comment]
we should do this at some point
